### ISSUE TITLE
F9 for \ref{ does not work on Windows

### ISSUE THREAD

**Issue by Ben30230** [vim-latex/vim-latex#214]
I followed the usage in the http://vim-latex.sourceforge.net/documentation/latex-suite/ls-completion-ref.html, But when I press F9, it told me that can not open C:/Users/XXX/AppData/Local/Temp/VIo6B5D.tmp. I found that  VIo6B5D.tmp doesn't exist at all. 

But \cite{ F9 works well : ) 

My environment is Windows 10 with python 3.10.4

**Comment by gerw** [comment]
Can you show the output of
````vim
:echo Tex_GetVarValue('Tex_UseOutlineCompletion')
:echo Tex_GetVarValue('Tex_UseSimpleLabelSearch')
:echo Tex_GetVarValue('Tex_ProjectSourceFiles')
:echo Tex_UsePython()
````
?

Does it tell you, in which file the error occurred?

**Comment by Ben30230** [comment]
they are respectively 1  0 nothing 0

**Comment by gerw** [comment]
Do you have python installed on your machine? (Try, e.g.,
````
echo system('python --version')
echo system('python3 --version')
````
)?

**Comment by Ben30230** [comment]
The first command: Python 3.10.4
The second command: nothing

**Comment by Ben30230** [comment]
Both showed: can not open the file C:/Users/XXX/AppData/Local/Temp/VIo4B44.tmp (same error as the original error when pressing F9)

**Comment by Ben30230** [comment]
I try to run "path_to_latex_suite/ftplugin/latex-suite/outline.py path_to_some_tex_file/file.tex" in a Linux like enviroment (git bash) and it works. 

I think something goes wrong with "shellescape".  I am trying to figure it out...

By the way, a few of my friends have the same problem with \ref{F9 , mabey this is a generic problem...

**Comment by Ben30230** [comment]
I found that 

:echo system('python "C:\Program Files (x86)\Vim\vim82\ftplugin\latex-suite\outline.py" C:\Users\Ben\Desktop\test\test.tex')
works

:echo system('python C:\Program Files (x86)\Vim\vim82\ftplugin\latex-suite\outline.py C:\Users\Ben\Desktop\test\test.tex')
doesn't work (the same error as previous)

Maybe the problem is the double quotes! But how can I fix it?

**Comment by Ben30230** [comment]
Here is all my thinking:

The installation of vim-latex is just replacing 5 folders in my vim installation path.  I don't know whether this matters.

I found the outline.py path may have trouble with ---C:\Program Files (x86)---(because of many spaces), so I re-install vim, but this time I change the installation path in -----C:\Vim\...------ 

This time when I press F9, in the __OUTLINE__ window, it shows that  -----''C:'----- is not a command (different from the previous problem)

I try to read the resources, maybe the problem goes with the path of outline.py file. But I don't know the syntax for this language. So I don't fix it yet.

Thanks a lot!

**Comment by gerw** [comment]
Can you please show the complete error message?

**Comment by gerw** [comment]
What is your setting of `shellslash`? Did you try to set/unset it?

**Comment by Ben30230** [comment]
In vimrc I have this sentence ---       set shellslash
when I press F9, the vim will show a embeded windows, the title is "__OUTLINE__" but the text it shows is that ---
''C:' is not an internal or external command, nor is it a runnable program or batch file.

When delete ---   set shellslash
when press F9 it will open the ouline.py file in VScode IDE. and then nothing happens.

**Comment by Ben30230** [comment]
This time I have changed the installation path for vim in C:/VIM

**Comment by Ben30230** [comment]
I found the problem !!!
In Makefile  I found that outline.py have been transfer to a executable file. So 

---path_to_latex_suite/ftplugin/latex-suite/outline.py C:\Users\Ben\Desktop\test\test.tex--- should works

but when I run it, it will just open it in vscode. So i change the default opening method from vscode to python for outline.py.


And I delete set shellslash then press F9 it works! Magic!

Thank you vvvvvvvery much!
